website/sidebars.js

```diff
@@ -80,81 +160,95 @@ const sidebars = {
-          id: 'devs-how-tos/how-to-use-oracles',
+          type: 'category',
+          label: 'Concepts',
+          collapsed: true,
+          items: [
+            {
+              type: 'autogenerated',
+              dirName: 'for-devs/concepts/token-bridge',
+            },
+          ],
         },
       ],
     },
     {
       type: 'category',
-      label: 'Concepts',
-      collapsed: true,
+      label: 'Migrate to Arbitrum',
+      collapsed: false,
       items: [
-        {
-          type: 'doc',
-          label: 'Arbitrum chains',
-          id: 'for-devs/concepts/public-chains',
-        },
         {
           type: 'category',
-          label: 'Arbitrum/Ethereum differences',
+          label: 'From Classic to Nitro',
+          collapsed: true,
           items: [
             {
               type: 'doc',
-              label: 'Arbitrum vs. Ethereum',
-              id: 'arbitrum-ethereum-differences',
+              label: 'Migrate from Classic to Nitro (architecture, contracts)',
+              id: 'migration/dapp_migration',
             },
             {
               type: 'doc',
-              label: 'Solidity support',
-              id: 'solidity-support',
+              label: 'Migrate from Classic to Nitro (state)',
+              id: 'migration/state-migration',
             },
+          ],
+        },
+        /** {
+          type: 'category',
+          label: 'From Ethereum (L1) to Arbitrum (L2)',
+          collapsed: true,
+          items: [
             {
               type: 'doc',
-              label: 'Block numbers and time',
-              id: 'time',
+              label: 'TODO (vibe-checking)',
+              id: 'migration/dapp_migration',
             },
           ],
         },
-        {
-          type: 'doc',
-          label: 'Cross-chain messaging',
-          id: 'for-devs/cross-chain-messsaging',
-        },
         {
           type: 'category',
-          label: 'Token bridge',
-          collapsed: false,
+          label: 'From Arbitrum (L2) to Orbit',
+          collapsed: true,
           items: [
             {
-              type: 'autogenerated',
-              dirName: 'for-devs/concepts/token-bridge',
+              type: 'doc',
+              label: 'TODO (vibe-checking)',
+              id: 'migration/dapp_migration',
             },
           ],
         },
         {
-          type: 'doc',
-          label: 'Precompiles',
-          id: 'for-devs/concepts/precompiles',
+          type: 'category',
+          label: 'From Polkadot to Arbitrum',
+          collapsed: true,
+          items: [
+            {
+              type: 'doc',
+              label: 'TODO (vibe-checking)',
+              id: 'migration/dapp_migration',
+            },
+          ],
         },
         {
-          type: 'doc',
-          label: 'Oracles',
-          id: 'for-devs/concepts/oracles',
-        },
+          type: 'category',
+          label: 'From Solana to Arbitrum',
```

Comment:
Something to consider for vNext -> migration guides targeted at ecosystems that need them (after validating the need via lightweight customer dev / discovery)